src/internal_events/datadog_traces.rs

```diff
@@ -14,7 +14,7 @@ impl InternalEvent for DatadogTracesEncodingError {
     fn emit(self) {
         error!(
             message = "Failed to encode Datadog traces.",
-            error = %self.message,
```

Comment:
Same here.

Reply:
Attempted to address here https://github.com/vectordotdev/vector/pull/14052/commits/138b1abbc17c16382d00f130e1f2d325da4f1135